devel/protobuf3-cpp-upstream/Portfile

```diff
@@ -15,12 +15,12 @@ legacysupport.newest_darwin_requires_legacy 15
 # NOTE: For a minor or major version number change, also
 # NOTE:   Revbump et, protobuf-c, mosh and py-onnx
```

Comment:
loops like this PR is missing some revbumps?

Reply:
This is a development subport tracking upstream with no dependents: https://ports.macports.org/port/protobuf3-cpp-upstream/details/
(If someone builds against it locally, they will need to revbump, but no need to do it for everyone.)

Reply:
ah, I see - this isn't the default for the typical user. Thanks for pointing that out; should the comment be removed then?

Reply:
Yeah, let me do that indeed.

Reply:
@reneeotten I have updated portfile comments. Does it look okay now?